apps/parsley/package.json

```diff
@@ -19,7 +19,7 @@
     "deploy:prod": "env-cmd -e production ts-node scripts/deploy/run-deploy",
     "deploy:do-not-use": ". ./scripts/deploy/deploy.sh",
     "dev": "env-cmd -e devLocal -r .env-cmdrc.local.json vite",
-    "evg-db-ops": "yarn --cwd='../..' evg-db-ops",
+    "evg-db-ops": "../../scripts/evg-db-ops.sh",
```

Comment:
Should this script be removed from the root `package.json`? I believe it would fail for the same reason that running from an app with `--cwd` failed.